### ISSUE TITLE
mouse keyboard BLE example

### ISSUE THREAD

**Issue by dartecne** [arduino-libraries/ArduinoBLE#269]
It would be really lovely to include a classic bluetooth Mouse Keyboard example as BLE peripherical!
I tried to make it, but after many hours trying I gave it up.
Thanks a lot for your work!

**Comment by henry63** [comment]
I totally agree that such a code exemple would be great. I've been trying to write code to read mouse report in boot mode for 3 weeks, without success.
